### DISCUSSION TITLE
Multer.rs new release

### DISCUSSION THREAD

**Discussion by rousan** [rwf2/multer#30] [General]

**Comment by gsserge** [comment]
Hi @rousan! Can you please make a new release? I'm trying to update `routerify-multipart` but I think this one should be released first with up-to-date Tokio and Hyper. Thanks!

**Reply by rousan** [reply]
Sure, @SergioBenitez, are there any pending issues to be solved? I think we should make a new release as soon as possible. @SergioBenitez, Please let me know.

**Reply by rousan** [reply]
@gsserge, a major version is released for multer.rs, `v2.0.0`.

**Reply by gsserge** [reply]
@rousan thank you very much!
